doc/src/build/zk_login.md

```diff
@@ -477,23 +477,31 @@ zkLogin can support providers that work with OpenID Connect built on top of the
 
 Traditional private key wallets demand users to consistently recall mnemonics and passphrases, necessitating secure storage to prevent fund loss from private key compromise.
 
-On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. A loss of ephemeral keys alone does not mean a loss of funds.
+On the other hand, a zkLogin wallet only requires an ephemeral private key storage with session expiry and the OAuth login flow with expiry. Forgetting ephemeral keys alone does not result in loss of funds, because a user can always sign in again to generate a new ephemeral key and a new ZK proof.
 
 ## How is zkLogin different from MPC or Multisig wallets?
 
 Multi-Party Computation (MPC) and Multisig wallets rely on multiple keys or distributing multiple key shares and then defining a threshold value for accepting a signature.
 
-zkLogin does not split any individual private keys, but ephemeral private keys are registered using nonce when the user authenticates with the OAuth provider. The primary advantage of zkLogin is that the user does not need to manage any persistent private key anywhere, not even with any private keys management techniques like MPC or Multisig.
+zkLogin does not split any individual private keys, but ephemeral private keys are registered using a fresh nonce when the user authenticates with the OAuth provider. The primary advantage of zkLogin is that the user does not need to manage any persistent private key anywhere, not even with any private keys management techniques like MPC or Multisig.
+
+Interestingly though, a zkLogin address automatically simulates a 2-of-2 setting, where the first part is user's OAuth account and the second is user's salt.
+
+Furthermore, because Sui native supports Multisig wallets, one can always include one or more zkLogin signers inside a Multisig wallet for additional security, such as using the zkLogin part as 2FA in k-of-N settings.
 
 ## Does losing my OAuth credential mean the loss of funds in the zkLogin wallet?
```

Comment:
I would suggest reframing this into two separate questions?

- If my OAuth account is compromised, what happens to my zkLogin address?
- If I lose access to my OAuth account, do I lose access to my zkLogin address?